lib/fastlane/plugin/revenuecat_internal/helper/revenuecat_internal_helper.rb

```diff
@@ -24,51 +33,59 @@ def self.replace_version_number(previous_version_number, new_version_number, fil
         end
       end
 
+      def self.determine_next_version_using_labels(repo_name, github_token, rate_limit_sleep)
+        old_version = Actions.sh("git describe --tags --abbrev=0").strip
```

Comment:
Not needed for this PR but one thing I've noticed is that sometimes I haven't fetched the latest tags from origin. Maybe we could do that before this. Again, not needed for this PR and I can do it later as well.